src/test/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherDefaultDueDateServiceImplTest.java

```diff
@@ -118,11 +103,18 @@ protected ParameterService buildMockParameterService(String numberOfDays) {
         return service;
     }
     
-    protected DateTimeService buildMockDateTimeService(Calendar currentCalendar) {
+    protected DateTimeService buildMockDateTimeService() {
         DateTimeService service = Mockito.mock(DateTimeService.class);
-        Mockito.when(service.getCurrentCalendar()).thenReturn(currentCalendar);
+        Mockito.when(service.getLocalDateNow()).thenReturn(java.time.LocalDate.now(Clock.systemDefaultZone()));
+        when(service.getSqlDate(any(LocalDate.class))).then(this::getSqlDate);
```

Comment:
I would suggest updating this method to create an instance of `TestDateTimeServiceImpl` and then use `Mockito.spy()` to create a spied version of it. That way, you can let the actual class handle most of the operations, and you can use one of the `Mockito.doXXX()` methods to override the behavior of specific service methods. This will allow the unit test to override the current date-time as before (though the test should be updated to use a `java.time` class instead of `Calendar`).